src/backer/cardaning.py

```diff
@@ -120,6 +121,41 @@ def flushQueue(self):
             self.timer = Timer(90, self.flushQueue)
             self.timer.start()
 
+    def publishEventv2(self, event: bytearray):
+        self.pending_kel = event
+        if not self.timer.is_alive():
+            self.timer = Timer(90, self.flushQueue)
+            self.timer.start()
+
+
+    def flushQueueV2(self):
```

Comment:
Can we just replace the old `flushQueue` and `publishEvent` with this code?

Reply:
updated 
ref here: 60b40541ac1ece5dff7ef05ea8725fb0c7342700